bundles/framework/timeseries/view/TimeSeriesRange/TimeseriesMetadataService.js

```diff
@@ -7,7 +7,7 @@ const DEFAULT_STYLE = {
         color: 'rgba(0, 0, 0, 0.8)'
     },
     fill: {
-        color: 'rgba(24, 219, 218, 0.5)'
+        color: 'rgba(24, 219, 218, 0)'
```

Comment:
Maybe just remove the fill object from style instead of making it transparent?

Reply:
Removing the fill will use a default fill color (a purplish color) defined somewhere else.